### DISCUSSION TITLE
Billing without a credit card

### DISCUSSION THREAD

**Discussion by jmartin6716** [vercel/community#3188] [Feature Question] [answered]
### Summary

Is there a way to do billing without a credit card? My company has changed its way of doing invoices and they are looking to get invoiced for Vercel services and then pay a more traditional way without a credit card. If someone from Vercel can reach out and help me with this, that would be awesome!  Thank you in advance.

### Steps to Reproduce

 Billing/Invoicing question

**Comment by swarnava** [comment, marked as answer]
Hi,

Invoice based payment is available only for Enterprise users with Annual payment. You can speak to our Sales team for more information on this: https://vercel.com/contact/sales

Unfortunately, we don't accept invoice based payment for Pro customer with monthly billing
